workspace/notebook/py_unit_tests_functions.json

```diff
@@ -776,6 +776,36 @@
 				],
 				"execution_count": null
 			},
+			{
+				"cell_type": "code",
+				"metadata": {
+					"jupyter": {
+						"source_hidden": false,
+						"outputs_hidden": false
+					},
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"def test_no_project_from_horizon_dropped(std_db: str, curated_db_migration: str, horizon_table:str, curated_table_migration:str):\n",
```

Comment:
The objective of this test is to make sure no record from horizon is dropped. I think this could be made a bit more generic. i.e by making the primary key a parameter. I believe for s51 advice, the primary key is advicenodeid/adviceId instead of caseReference. 

Something like this?
`def test_no_record_from_horizon_dropped(std_db: str, curated_db_migration: str, horizon_table:str, curated_table_migration:str, horizon_pk: str, curated_pk: str)`

Reply:
And then we can pass `caseNodeId` as a horizon_pk from s51 advice unit test notebook, and `caseReference` from nsip_project unit test notebook